scripts/sync_repo_files.sh

```diff
@@ -96,6 +96,15 @@ check_license() {
   echo "$1" | grep --quiet --no-messages --ignore-case 'Apache License'
 }
 
+check_go() {
+  local org_repo
+  local default_branch
+  org_repo="$1"
+  default_branch="$2"
+
+  curl -sLf -o /dev/null "https://raw.githubusercontent.com/${org_repo}/${default_branch}/go.mod"
```

Comment:
hm, that might be not enough. There might be repo with Go but not in root repo

Reply:
wonder if there is more reliable way 🤔

Reply:
I assume that if go.mod is not at the root, we should not sync this and it would require more parameters, so that's fine to me.

Reply:
Then it never runs in CI.

Reply:
Well, checking if we're in CircleCI _and_ the Actions file is present.